target/linux/ramips/image/mt7621.mk

```diff
@@ -215,6 +240,26 @@ define Device/asus_rt-n56u-b1
 endef
 TARGET_DEVICES += asus_rt-n56u-b1
 
+define Device/beeline_smartbox-flash
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  DEVICE_VENDOR := Beeline
+  DEVICE_MODEL := SmartBox Flash
+  IMAGE_SIZE := 55808k
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | uImage none \
+	| pad-to 16805860 | beeline-sb-flash-header
```

Comment:
I would prefer using `|` before the line break to comply with the convention.

Reply:
@mans0n fixed. thanks.